src/main/resources/reference.conf

```diff
@@ -201,10 +237,10 @@ cassandra-query-journal {
   
   # How many events to fetch in one query (replay) and keep buffered until they
   # are delivered downstreams.
-  max-buffer-size = 100
+  max-buffer-size = 10000
```

Comment:
Also note this is dependent on fetch-size, because if fetch-size > buffer-size then we can easily have more than this specifies in the buffer after a fetch. Especially when a separate query actor is used, we could (at some moment in time) have fetch-size elements in ResultSet and max-buffer-size elements in the actor?